prow/config.yaml

```diff
@@ -3185,6 +3176,76 @@ postsubmits:
           secretName: service-account
 
 periodics:
+- name: ci-cri-containerd-build
+  interval: 30m
+  agent: kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/gcloud-in-go:0.4
+      args:
+      - "--repo=github.com/kubernetes-incubator/cri-containerd=master"
+      - "--root=/go/src"
+      env:
+      - name: GOOGLE_APPLICATION_CREDENTIALS
+        value: /etc/service-account/service-account.json
+      volumeMounts:
+      - name: service
+        mountPath: /etc/service-account
+        readOnly: true
+    volumes:
+    - name: service
+      secret:
+        secretName: service-account
+
+- name: ci-cri-containerd-node-e2e
+  interval: 1h
+  agent: kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20171008-cd94f30a-master
+      args:
+      - --root=/go/src
+      - "--git-cache=/root/.cache/git"
```

Comment:
I don't see `git-cache` is used in the code, do we still need the ssd cache?

Reply:
it's in https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap.py#L225, but you don't have to use it